src/bosh-stemcell/spec/os_image/ubuntu_trusty_spec.rb

```diff
@@ -303,8 +303,14 @@
   end
 
   context 'PAM configuration' do
-    describe file('/lib/x86_64-linux-gnu/security/pam_cracklib.so') do
-      it { should be_file }
+    if Bosh::Stemcell::Arch.ppc64le?
```

Comment:
dont forget about updating this for a looser check on where pam_cracklib.so lives, to avoid branching.